dds/idl/be_produce.cpp

```diff
@@ -206,6 +206,7 @@ void postprocess(const char* fn, ostringstream& content,
     }
     if (which == BE_GlobalData::STREAM_H) {
       out << "#include \"dds/DCPS/Definitions.h\"\n";
+      out << "#include \"dds/DdsDcpsC.h\"\n";
```

Comment:
This file includes again DdsDcpsDomainC.h which defines for example the DomainParticipant, we need to expose that to the user